azkaban-common/src/main/java/azkaban/executor/JdbcExecutorLoader.java

```diff
@@ -1364,6 +1382,52 @@ public String handle(ResultSet rs) throws SQLException {
     }
   }
 
+  private static class FetchRecentlyFinishedFlows implements
```

Comment:
Would FetchRecentlyFinishedFlowsHandler be a better name? though I know the handler classes in this file don't have this convention. What do you think?